assets/onboarding/features/confirmation-modal.jsx

```diff
@@ -0,0 +1,63 @@
+import { List } from '@woocommerce/components';
+import { Button, Modal } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * @typedef  {Object} Feature
+ * @property {string} title                          Feature title.
+ * @property {string} description                    Feature description.
+ * @property {string} [confirmationExtraDescription] Extra description that appears only in confirmation modal.
+ */
+/**
+ * Features confirmation modal.
+ *
+ * @param {Object}    props
+ * @param {Feature[]} props.features  Features list.
+ * @param {Function}  props.onInstall Callback to install the features.
+ * @param {Function}  props.onSkip    Callback to skip the installation.
+ */
+const ConfirmationModal = ( { features = [], onInstall, onSkip } ) => (
+	<Modal
+		className="sensei-onboarding__features-confirmation-modal"
+		title={ __(
+			'Would you like to install the following features now?',
+			'sensei-lms'
+		) }
+		isDismissible={ false }
+	>
+		<List
+			items={ features.map(
+				( { title, description, confirmationExtraDescription } ) => ( {
+					title,
+					content: `${ description } ${ confirmationExtraDescription }`,
```

Comment:
We might need to check for `undefined` here.

<img width="464" alt="Screen Shot 2020-05-12 at 16 36 32" src="https://user-images.githubusercontent.com/842193/81737655-f4655980-9487-11ea-98b6-47a09d398523.png">

This would be a good thing to add a test for! 🙂

Reply:
Sure! Fixed here: https://github.com/Automattic/sensei/pull/3121/commits/037568f6128de91d012b6dcda950aeedefca8bc1